common/metrics/metric_defs.go

```diff
@@ -1145,6 +1145,9 @@ var (
 		"schedule_action_delay",
 		WithDescription("Delay between when scheduled actions should/actually happen"),
 	)
+	NonRetryableTasks = NewCounterDef(
```

Comment:
nit: move it to matching section L1009
```suggestion
	TaskDroppedCounter = NewCounterDef(
```

Reply:
Actually looks like this def is not used?

Reply:
sorry - I intended on emitting this def as a metric later on in my code. Changed to reflect that.